pkg/apis/core/v1alpha1/federatedcluster_types.go

```diff
@@ -61,7 +63,10 @@ type FederatedClusterStatus struct {
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// FederatedCluster
+// FederatedCluster configures federation to be aware of a Kubernetes cluster
+// from the cluster-registry and provides a Kubeconfig for federation to use to
```

Comment:
Do we need to mention this `provides a Kubeconfig` at here again? I think it already been mentioned in `FederatedClusterSpec`?

Reply:
Does not seem like a dead repetition.